src/banman.cpp

```diff
@@ -25,7 +25,7 @@ BanMan::BanMan(fs::path ban_file, CClientUIInterface* client_interface, int64_t
         SweepBanned();            // sweep out unused entries
 
         LogPrint(BCLog::NET, "Loaded %d banned node ips/subnets from banlist.dat  %dms\n",
-            banmap.size(), GetTimeMillis() - nStart);
```

Comment:
Hungarian or snake_case, pick one :)

Reply:
Fixed in 9bad9fa1168ec53e92bb8b2df0b5e9d578344a2f